src/Runner.jl

```diff
@@ -351,6 +351,22 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
                 min_macos_version_flag(p),
             ])
         end
+        if Sys.isfreebsd(p)
+            # clang on FreeBSD does not have `SYSROOT/usr/local` (which in our build
+            # environments is a symlink to $includedir = "$(prefix)/include") in its
+            # default list of include search directories. As a result, quite some
+            # builders that work fine everywhere else need to add something like
+            # `-I${includedir}` to their CPPFLAGS when building for FreeBSD.
+            #
+            # To remove the need for this, we simply add it here. We use `-isystem`
+            # instead of `-I` to mimic what is done in clang builds for other operating
+            # systems. It is still not a perfect match, as the order of the default
+            # search directories differs, but this should not matter in practice.
+            #
+            # See also https://github.com/JuliaPackaging/Yggdrasil/issues/3949 and
+            # https://github.com/JuliaPackaging/Yggdrasil/pull/3969 for further details.
+            append!(flags, ["-isystem", "/workspace/destdir/include"])
```

Comment:
Actually, considering what I write above, perhaps the simple way to avoid hardcoding the value of $prefix here would be to do this (and thus really mirroring what is the case elsewhere):
```suggestion
            append!(flags, ["-isystem", "/usr/local/include"])
```
Thoughts?

Reply:
```suggestion
            append!(flags, ["-isystem", raw"${includedir}"])
```
?

Reply:
Nice!